src/TextUI/Command.php

```diff
@@ -68,6 +68,8 @@ class Command
     protected $longOptions = [
         'atleast-version='          => null,
         'bootstrap='                => null,
+        'cache-result'              => null,
+        'cache-result-file='        => null,
```

Comment:
I am not sure if we really want this to be configurable using CLI options. If caching could only be turned on through `phpunit.xml` then the hidden cache file could be placed next to it.

Reply:
> If caching could only be turned on through `phpunit.xml`

`cache-result` is to enable cache via CLI

Reply:
I understand that. What I am not sure about is the value of being able to turn on caching on the commandline.

Reply:
Hmmm yes. I haven't really turned it off anymore since it became stable to use. 🤔